eng/versioning/version_client.txt

```diff
@@ -408,8 +408,8 @@ com.azure.tools:azure-sdk-build-tool;1.0.0-beta.1;1.0.0-beta.2
 # In the pom, the version update tag after the version should name the unreleased package and the dependency version:
 # <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
 
-unreleased_com.azure:azure-core-amqp;2.9.0-beta.1
-unreleased_com.azure:azure-core;1.37.0-beta.1
+unreleased_com.azure:azure-core-amqp;2.8.3
+unreleased_com.azure:azure-core;1.37.0
```

Comment:
After these versions are released we should make sure to remove these entries from this file and update the POMs where they were used to use the `dependency` version instead.

Reply:
That said, I'm not sure if the release pipeline is going to succeed for a dependency marked as unreleased. We might want to replace the tags in POMs with `current` instead.

Reply:
You are right, release pipeline will not succeed if we have these around and should be removed in the Increment PR.